.github/ISSUE_TEMPLATE/bug-report.md

```
@@ -0,0 +1,36 @@
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A clear and concise description of how the code performed w.r.t expectations.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Additional details**
+Add any other context or screenshots about the feature request here.
+
+**Potential internship candidates**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)